site/docs/providers/deepseek.md

```diff
@@ -41,23 +41,25 @@ providers:
 
 :::note
 
-The API model names are aliases that automatically point to the latest versions: `deepseek-chat` points to DeepSeek-V3-0324 and `deepseek-reasoner` points to DeepSeek-R1-0528.
+As of September 29, 2025, both `deepseek-chat` and `deepseek-reasoner` have been upgraded to use **DeepSeek-V3.2-Exp**, featuring significant cost reductions (50% lower pricing) and improved efficiency through DeepSeek Sparse Attention (DSA) technology.
 
 :::
 
-### deepseek-chat (DeepSeek-V3)
+### deepseek-chat (DeepSeek-V3.2-Exp)
 
+- Non-thinking mode of DeepSeek-V3.2-Exp
 - General purpose model for conversations and content
 - 64K context window, 8K output tokens
-- Input: $0.07/1M (cache), $0.27/1M (no cache)
-- Output: $1.10/1M
+- Input: $0.028/1M (cache hit), $0.28/1M (cache miss)
```

Comment:
It would really cool if you got the cost calculation into this PR. You can look at how providers like xai and togetherai are implemented as an example